src/prisma.ts

```diff
@@ -1,3 +1,14 @@
 import { PrismaClient } from '@prisma/client'
 
-export const prisma = new PrismaClient()
+const prisma = new PrismaClient()
+{
+  //This scope is only to check whether the table in DB exist.
+  //If table does not exist execute "pnpm prisma db push".
+  try {
+    await prisma.messageReportCount.findFirst()
+  } catch (error) {
+    console.log(error)
```

Comment:
Can we log the exact instruction in `README.md` in here?

Reply:
Actually, the same instruction about running command 
```
pnpm prisma db push
```
is also in README.md